apis/duck/v1alpha1/conditions_types.go

```diff
@@ -119,6 +119,32 @@ type KResourceStatus struct {
 	Conditions Conditions `json:"conditions,omitempty"`
 }
 
+var condSet = NewBatchConditionSet()
```

Comment:
this will not do what you want for all things that might use this. 
I have a competing PR to do this same thing:  https://github.com/knative/pkg/pull/80

Reply:
I copied over from Build :)

It'd probably make sense to remove the two methods depending on `condSet` and maybe use `condSet.Manage(krs).GetCondition(t)` directly - I just need `GetCondition(t)`.

Reply:
Done!